packages/eddsa-pcd/src/EdDSAPCD.ts

```diff
@@ -174,6 +363,7 @@ export const EdDSAPCDPackage: PCDPackage<
   deserialize
 };
 
+/** @ignore */
```

Comment:
I put the `@ignore` on top since the method was not used inside the `EdDSAPCD.ts` file as remainder to move it outside (eg., `utils` as you suggested) and punt back to the next PR, my bad!

Reply:
I will add a comment in the next commit.